src/Form/BasicOverview.php

```diff
@@ -38,14 +38,15 @@ public function buildForm(array $form, FormStateInterface $form_state, UserInter
     $enabled = isset($user_tfa['status']) && $user_tfa['status'] ? TRUE : FALSE;
 
     if (!empty($user_tfa)) {
+      $date_formatter = \Drupal::service('date.formatter');
       if ($enabled) {
-        $status_text = t('Status: <strong>TFA enabled</strong>, set !time. <a href="!url">Disable TFA</a>', array(
-          '!time' => format_date($user_tfa['saved']),
-          '!url' => URL::fromRoute('tfa_basic.tfa.disable', ['user' => $user->id()])->toString()
+        $status_text = t('Status: <strong>TFA enabled</strong>, set @time. <a href=":url">Disable TFA</a>', array(
+          '@time' => $date_formatter->format($user_tfa['saved']),
+          ':url' => URL::fromRoute('tfa_basic.tfa.disable', ['user' => $user->id()])->toString()
```

Comment:
When logging messages ':variable' is used for href attribute so I just followed the same thing here.